src/js/common/helpers/login-helpers.js

```diff
@@ -12,11 +13,22 @@ export function handleVerify(verifyUrl) {
 }
 
 export function getUserData(dispatch) {
+  
   fetch(`${environment.API_URL}/v0/user`, {
     method: 'GET',
     headers: new Headers({
       Authorization: `Token token=${sessionStorage.userToken}`
     })
+  }).catch(res => {
```

Comment:
I'm curious about the reasoning for adding this. It looks like it changes how this promise works a little and might be better at the end of the chain.

Reply:
This was added in the interest of being consistent, but if it's not achieving that goal, I will remove.